pkgs/applications/audio/mopidy/ytmusic.nix

```diff
@@ -16,7 +16,12 @@ python3Packages.buildPythonApplication rec {
     python3Packages.pytube
   ];
 
-  doCheck = false;
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace 'ytmusicapi>=0.19.1,<0.20.0' 'ytmusicapi>=0.19.1'
+  '';
+
+  doCheck = true;
```

Comment:
Tests are enables automatically for Python packages. However
> Ran 0 tests in 0.000s
because the tests are not shipped on PyPI. We could use `fetchFromGitHub` instead.

Reply:
Yes, we could, but I do not use the package myself, nor do I maintain it or plan to